extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpAnalytics} from '../../../../build/all/v0/amp-analytics-0.1.max';
+import {adopt} from '../../../../src/runtime';
+import * as sinon from 'sinon';
+
+adopt(window);
+
+describe('amp-analytics', function() {
+
+
+  function getAnalyticsTag(config, attrs) {
+    config = JSON.stringify(config);
+    const el = document.createElement('amp-analytics');
+    el.textContent = config;
+    const analytics = new AmpAnalytics(el);
+    analytics.firstAttachedCallback();
```

Comment:
Since AmpAnalytics listens to the real events, shouldn't you also remove it from DOM or deconstruct somehow to avoid listener leaks and tinted tests?